crates/biome_cli/src/execute/traverse.rs

```diff
@@ -45,8 +42,6 @@ pub(crate) fn traverse(
     cli_options: &CliOptions,
     mut inputs: Vec<OsString>,
 ) -> Result<TraverseResult, CliDiagnostic> {
-    init_thread_pool();
```

Comment:
Why did we remove this function?

Reply:
Because we can only initialize the thread pool once, and the workspace server does that now. It's a bit of a bummer, although I hope that eventually the CLI won't need to do traversal on its own anymore.